### DISCUSSION TITLE
ScriptReference/Screen-dpi

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#350] [Notes]
# ScriptReference/Screen-dpi

https://docs.unity3d.com/ScriptReference/Screen-dpi.html

**Comment by docscomments** [comment]
**Comment by `@SnugglePilot`:**  

Note that Screen.DPI returns the logical DPI on windows, which is 96 multiplied by the windows display properties "scaling factor".  
Detailed in [this forum thread](https://forum.unity.com/threads/is-screen-dpi-supported.192380/).

**Comment by SoylentGraham** [comment]
On ios, this is the device's dpi, and is NOT affected by the fixed-dpi, which changes the scale of the metal view.
Screen.width & Screen.height DO reflect the metal view's dimensions.
